Clients.ts

```diff
@@ -155,4 +155,15 @@ export default class Clients {
 
     }
 
+    async get_datatable_details(datatable_name: string, clientId: number): Promise<any> {
+        let search_query_obj = {
```

Comment:
search_query_obj should always be passed as argument to the function so that it can be customized by the code which is calling this function